### PR TITLE
#1

### PR DESCRIPTION
**Pull request by nhatnam113**

### CODE REVIEW

File: play2048.py [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
     # 2. uses the parameter 'times' to determine how many times your code will try to play the game
     # 3. for each 'time', press these keys in this order: UP, RIGHT, DOWN, LEFT
     # 4. print the final score after all tries to the screen 
-    
\ No newline at end of file
+    browser = webdriver.Firefox()
+    browser.get('https://gabrielecirulli.github.io/2048/')
+
+    htmlElem = browser.find_element_by_tag_name('html')
+
```
Comment (halfaro1) [inline play2048.py]: @nhatnam113 what about the score element?

Comment (nhatnam113) [inline play2048.py]: I see what I'm missing
Thank you very much professor

File: play2048.py [hunk 2]
```diff
@@ -1,10 +1,26 @@
 #TODO: Import the module that will allow you to use Selenium
+from selenium import webdriver
 #TODO: Import the module that will allow you to use the up, down, left, and right keys on your keyboard
+from selenium.webdriver.common.keys import Keys
 
 def play2048( times ):
     #TODO: write code in this function that:
     # 1. opens a game of 2048 from the URL: https://gabrielecirulli.github.io/2048/
     # 2. uses the parameter 'times' to determine how many times your code will try to play the game
```
Comment (halfaro1) [inline play2048.py]: @nhatnam113 what happened to using this parameter `times`?

File: play2048.py [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
\ No newline at end of file
+    browser = webdriver.Firefox()
+    browser.get('https://gabrielecirulli.github.io/2048/')
+
+    htmlElem = browser.find_element_by_tag_name('html')
+    scoreElem = browser.find_element_by_class_name('score-container')
+
+    moves = 0
+    while moves < times:
+        htmlElem.send.keys(Keys.UP)
```
Comment (halfaro1) [inline play2048.py]: @nhatnam113 isn't the function `send_keys`? This shouldn't work on your local machine are you getting warnings/errors in Python IDLE?

**Comment by halfaro1** [comment]
@nhatnam113 
Fantastic work :+1:
I'm marking this pull request as closed, and labeling it as `graded` while I finish grading these projects. When I'm done grading all the projects, and your grade should be on Blackboard, I'll change the tag to `grade updated on blackboard`

**Pull request closed without merging**